pkg/sysinfo/sysinfo_linux.go

```diff
@@ -13,82 +11,81 @@ import (
 
 // New returns a new SysInfo, using the filesystem to detect which features the kernel supports.
 func New(quiet bool) *SysInfo {
+	// Check if Devices cgroup is mounted, it is hard requirement for container security.
+	if _, err := cgroups.FindCgroupMountpoint("devices"); err != nil {
```

Comment:
Agreed, I can change this to return an error.
This originally comes from #11882